tests/js/testUtils.js

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2017 OCAD University
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/waharnum/sjrk-storyTelling/master/LICENSE.txt
+*/
+
+/* global fluid, sjrk */
+
+(function ($, fluid) {
+
+    "use strict";
+
+    sjrk.storyTelling.testUtils = {};
+    sjrk.storyTelling.testUtils.changeForm = function (component, selector, value) {
```

Comment:
How about `sjrk.storyTelling.testUtils.changeFormElement` as the name here instead? 

Can we also describe what the function does, and why it's a necessary test utility function in light of how we're using `gpii-binder` and need to make sure the change event triggers so that the form value is relayed to the model?